airflow/sensors/time_sensor.py

```diff
@@ -72,13 +73,8 @@ def __init__(self, *, target_time: datetime.time, **kwargs) -> None:
 
         self.target_datetime = timezone.convert_to_utc(aware_time)
 
-    def execute(self, context: Context) -> NoReturn:
-        trigger = DateTimeTrigger(moment=self.target_datetime)
+    def execute(self, context: Context) -> None:
```

Comment:
same here why we are changing NoReturn to None?

Reply:
Added it